doc/R.nvim.txt

```diff
@@ -1843,64 +1843,96 @@ See https://github.com/R-nvim/R.nvim/wiki/Remote-access
 
 ------------------------------------------------------------------------------
 6.33. View a data.frame or matrix                             *R.nvim-df-view*
-                                                              *csv_app*
-                                                              *csv_delim*
 
 
 You can see data.frames and matrices in external applications with
-<LocalLeader>rv. By default, R.nvim will generate a `tsv` file from the
-`data.frame` or `matrix` and display it in a new tab. But you can change this
-by setting the value of `csv_app` which must be a string.
-
-If `csv_app` begins with "terminal:", R.nvim will open a new tab with the
+<LocalLeader>rv. By default, R.nvim will generate a `tsv` file from the whole
+`data.frame` or `matrix` and paste its contents it in a new tab. But you can
+change this by setting the values of the `view_df` elements in your config.
+The default values are:
+>lua
+   view_df = {
+      open_app = "",  -- How to open the CSV
+      how = "tabnew", -- How to display the data if doing it within Neovim
+      csv_sep = "",   -- Field separator to be used when saving the CSV.
+      n_lines = -1,   -- Number of lines to save in the CSV (0 for all lines).
+      save_fun = "",  -- R function to save the data.frame in a CSV file
+      open_fun = "",  -- R function to open the data.frame directly
+                      -- (no conversion to CSV needed)
+   },
+<
+If `open_app` begins with "terminal:", R.nvim will open a new tab with the
```

Comment:
We are talking about TSV and then all the options are about CSV. Maybe make it clearer?